src/components/function-summary/value.tsx

```diff
@@ -77,7 +77,16 @@ const TupleResult = ({ tuple, isPoxAddr, btc }: any) => {
   );
 };
 
-const getValue = (arg: { name: string; type: any; repr: any; value: any }, btc: null | string) => {
+const isJSONString = (str: string) => {
+  try {
+    JSON.parse(str);
+    return true;
+  } catch (error) {
+    return false;
+  }
+};
+
+const getValue = (arg: any, btc: null | string) => {
```

Comment:
Please add unit tests to `getValue` function that cover both cases (arg.repr is/isn't json).